### DISCUSSION TITLE
storage/hwameistor/intro/ldm/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1664] [Announcements]
# storage/hwameistor/intro/ldm/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/storage/hwameistor/intro/ldm/

**Comment by liyuerich** [comment]
git clone https://github.com/hwameistor/local-disk-manager.git 

使用这个命令无法拉代码，已经不存在

**Comment by liyuerich** [comment]
进入 repo 对应的目录

第一步clone已经不对， 后续目录也不对。 后续的步骤都需要根据最新的目录，路径来更改

**Comment by liyuerich** [comment]
查看 LocalDisk 信息， 复制会把以下信息都复制上，执行的时候会报错
查看命令只要提供 kubectl get localdisk 即可

$ kubectl get localdisk
10-6-118-11-sda    10-6-118-11       Unclaimed
10-6-118-11-sdb    10-6-118-11       Unclaimed

**Comment by liyuerich** [comment]
通过 kuebctl get localdisk <name> -o yaml 查看更多关于某块磁盘的信息。

kuebctl 拼写错误，导致执行命令失败。 应该为 kubectl

**Comment by liyuerich** [comment]
该命令用于获取集群中磁盘资源信息，获取的信息总共有四列，含义分别如下：

根据命令， 没有claim这一列

**Comment by liyuerich** [comment]
kubectl apply -f deploy/samples/hwameistor.io_v1alpha1_localdiskclaim_cr.yaml

error: the path "deploy/samples/hwameistor.io_v1alpha1_localdiskclaim_cr.yaml" does not exist
